lib/util.js

```diff
@@ -346,7 +346,8 @@ function formatValue(ctx, value, recurseTimes) {
   // Provide a hook for user-specified inspect functions.
   // Check that value is an object with an inspect function on it
   if (ctx.customInspect && value) {
-    const maybeCustomInspect = value[customInspectSymbol] || value.inspect;
+    const maybeCustomInspect = value[customInspectSymbol] ||
```

Comment:
I don't believe so, as Symbols are expected in all ES platforms. For example es's `toString` invokes `Symbol.toStringTag`